envsec/pkg/envcli/auth.go

```diff
@@ -80,39 +81,15 @@ func whoAmICmd() *cobra.Command {
 		Short: "Show the current user",
 		Args:  cobra.ExactArgs(0),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			client, err := newAuthClient()
-			if err != nil {
-				return err
-			}
-
-			tok, err := client.GetSession(cmd.Context())
-			if err != nil {
-				return fmt.Errorf("error: %w. Run `envsec auth login` to log in", err)
-			}
-			idClaims := tok.IDClaims()
-
-			fmt.Fprintf(cmd.OutOrStdout(), "Logged in\n")
-			fmt.Fprintf(cmd.OutOrStdout(), "User ID: %s\n", idClaims.Subject)
-
-			if idClaims.OrgID != "" {
-				fmt.Fprintf(cmd.OutOrStdout(), "Org ID: %s\n", idClaims.OrgID)
-			}
-
-			if idClaims.Email != "" {
-				fmt.Fprintf(cmd.OutOrStdout(), "Email: %s\n", idClaims.Email)
-			}
-
-			if idClaims.Name != "" {
-				fmt.Fprintf(cmd.OutOrStdout(), "Name: %s\n", idClaims.Name)
-			}
-
-			if flags.showTokens {
-				fmt.Fprintf(cmd.OutOrStdout(), "Access Token: %s\n", tok.AccessToken)
-				fmt.Fprintf(cmd.OutOrStdout(), "ID Token: %s\n", tok.IDToken)
-				fmt.Fprintf(cmd.OutOrStdout(), "Refresh Token: %s\n", tok.RefreshToken)
-			}
-
-			return nil
+			return (&envsec.Envsec{
+				APIHost: build.JetpackAPIHost(),
+				Auth: envsec.AuthConfig{
+					ClientID: envvar.Get("ENVSEC_CLIENT_ID", build.ClientID()),
+					Issuer:   envvar.Get("ENVSEC_ISSUER", build.Issuer()),
+				},
+				IsDev:  build.IsDev,
+				Stderr: cmd.ErrOrStderr(),
```

Comment:
this looks copy-pasta from `envcli/init.go`. Can you pull into a helper function?